wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -447,9 +466,7 @@ def run(self):
                 #         'Slug': 'your-model'
                 #   }
                 # This will return 'your-model' and can now be searched for as model.objects.get(slug='your-model')
-                unique_identifier = record_fields.get(
-                    airtable_unique_identifier_column_name, None
-                )
+                unique_identifier = record_fields.get(airtable_unique_identifier_column_name, None)
                 was_updated = self.update_object_by_uniq_col_name(
```

Comment:
Isn't the hook run in this function as well? In that case, won't it potentially run multiple times? I think it would be good to try and reduce the number of places we call this - the path through the import seems quite complex at the moment

Reply:
I could not agree more. This import class is getting pretty unruly at this point. The hook is run 2 other times in this function, but they `continue` in the loop afterwards so they exit it the loop early. The 3rd time it _could_ run is at the very end of the loop.